doc/programming_guide/ffd/overview.rst

```diff
@@ -13,6 +13,13 @@ The examples using the microphone array as the audio source include an audio pip
     #. Noise Suppressor (NS)
     #. Adaptive Gain Control (AGC)
 
+The FFD examples provide several options to inform the host of a possible intent detected by the intent engine. The device can notify the host by:
+
+  - sending the intent ID over UART interface upon the detection
```

Comment:
`over UART` -> `over a UART`

Also a similar change on line 19.

Reply:
Fixed in https://github.com/lucianomartin/sln_voice/pull/1/commits/2e4c50e53433b45b7e70b0990856266d7f306f5a.